[Bug]: Error TS1308: 'await' expressions are only allowed within async functions and at the top levels of modules.

### Describe the bug

When openning a new PR, one Github Action is failing due to a typescript error (see: https://github.com/MetaMask/metamask-extension/actions/runs/13548490023/job/37865999390?pr=30582)

we need to add async to the beggin of the function.



### Expected behavior

No typescript error due to the missing async keyword

`
// This function adds the "needs-triage" label to the issue if it doesn't have it
async function addNeedsTriageLabelToIssue(
  octokit: InstanceType<typeof GitHub>,
  issue: Labelable,
): Promise<void> {
  await addLabelToLabelable(octokit, issue, needsTriageLabel);
}
`



### Screenshots/Recordings

![Image](https://github.com/user-attachments/assets/7b0907ad-992c-4258-bdaf-74592de0169c)

### Steps to reproduce

1. Open a PR
2. Check the failling pipeline

### Error messages or log output

```shell
Error: .github/scripts/check-template-and-add-labels.ts(272,3): error TS1308: 'await' expressions are only allowed within async functions and at the top levels of modules.
```

### Detection stage

In production (default)

### Version

7.10.1

### Build type

None

### Browser

Chrome

### Operating system

MacOS

### Hardware wallet

_No response_

### Additional context

_No response_

### Severity

- Non critital bug
- Pull request creation

--- a/.github/scripts/check-template-and-add-labels.ts
+++ b/.github/scripts/check-template-and-add-labels.ts
@@ -265,7 +265,7 @@ function extractReleaseVersionFromBugReportIssueBody(
 }
 
 // This function adds the "needs-triage" label to the issue if it doesn't have it
-function addNeedsTriageLabelToIssue(
+async function addNeedsTriageLabelToIssue(
   octokit: InstanceType<typeof GitHub>,
   issue: Labelable,
 ): Promise<void> {

